testsuite/MDAnalysisTests/converters/test_openmm_parser.py

```diff
@@ -96,6 +96,25 @@ def test_segids(self, top):
         else:
             assert top.segids.values == []
 
+    def test_elements(self, top):
+        if hasattr(top, 'elements'):
```

Comment:
rather than looking for the attributes of the topology, it would be safer (and probably a better test) to look at whether or not "elements" is in "expected_attrs".

Also please note that adding these only tests the present of elements, you should also be adding tests for when there aren't elements (or partial elements, if that's something we want to support).